activiti-cloud-services-query/activiti-cloud-services-query-graphql-ws/src/main/java/org/activiti/cloud/services/query/qraphql/ws/transport/GraphQLBrokerSubProtocolHandler.java

```diff
@@ -112,7 +135,7 @@ else if (GraphQLMessageType.CONNECTION_TERMINATE.equals(payload.getType())) {
 					if (sent) {
 						if (isConnect) {
 							Principal user = headerAccessor.getUser();
-							if (user != null && user != session.getPrincipal()) {
+							if (user != null && user == session.getPrincipal()) {
```

Comment:
Is that normal that we are using `==` instead of `equals(...)` here?

Reply:
Yes, it comares that two references are of same object.